### ISSUE TITLE
Accessibility bugs in calamares

### ISSUE THREAD

**Issue by ghost** [calamares/calamares#523]
Jira issue originally created by user coffeeking:

the current state of calamares accessibility is next to nil. Screen reading software, such as orca, cannot see the window or the controls in that window. Since calamares is a qt5 application, accessibility shouldn't be too difficult to fix, and you no longer need to explicitly export the qt_accessibility variable. I will work with you guys to help fix this i don't expect this to be fixed for me. To begin with, you might try emailing joanmeri diggs at <jdiggs@igalia.com> to see if she'll try using calamares with orca to see if she can determine what might need to be done to fix it. This is very important and thus marked as major because I am a co developer of a manjaro based distribution targeted specifically at disabilities of all kinds called sonar <http://www.sonargnulinux.com> and we're trying to switch to calamares from manjaro's old installer.

**Comment by gladhorn** [comment]
https://codereview.qt-project.org/#/c/205196/ will most likely fix the issue when running the application as root. I don't want to encourage that, but it may be useful as a stop gap.

**Comment by adriaandegroot** [comment]
Dup by #1076 @mhussaincov

**Comment by entraide-net** [comment]
Half of 2019, 3 years later and still not accessible, you should be at shame ...

**Comment by philmmanjaro** [comment]
@entraide-net: accessibility is currently not our focus. It is still on our todo list, but with low priority. Currently we have to stabilize the install framework again.

**Comment by tuxayo** [comment]
As a humble contribution, a bounty has been created: [![Bountysource](
https://api.bountysource.com/badge/issue?issue_id=45108332
)](
https://www.bountysource.com/issues/45108332-accessibility-bugs-in-calamares
)

**Comment by mhussaincov93** [comment]
hi,
is there any news on this?
it really would be greate if this could be sorted out once and for all,
it would make distributions accessible such as manjaro because they use the calamares installer.
currently because calamares is not accessible, we are unable to install distributions like manjaro

**Comment by demmm** [comment]
This feature request is now well over 6 years old, no PR ever submitted for it, plus it can't be implemented until Calamares no longer runs as root.  Time to close this request now.  Better create a new request if/when Calamares can use Orca as user.

**Comment by entraide-net** [comment]
@demmm Yeah close an unfixed bug claiming it fixed, what a good idea :

https://youtu.be/WWx07b4IVbo?si=A00OQ75ne7qIITuG&t=1960

**Comment by demmm** [comment]
Please don't make false statements, nobody claimed it was fixed.
